### PR TITLE
Fix set typeid

### PR DESCRIPTION
**Pull request by kriskowal**
Fixes a bug identified by @Willyham

### CODE REVIEW

File: test/set.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ var thrift = new Thrift({source: source});
 test('Struct with set rw', testRW.cases(thrift.Bucket.rw, [
 
     [new thrift.Bucket({asArray: [1, 2, 3]}), [
-        0x0f,                   // type:1      -- 15, struct
+        0x0e,                   // type:1      -- 14, set
```
Comment (kriskowal) [inline test/set.js]: It is unclear to me how this ever worked with the struct typeid. I expected it would have been list.

**Comment by kriskowal** [comment]
r @Raynos @ShanniLi @malandrew

**Comment by ShanniLi** [comment]
:+1:

**Comment by Raynos** [comment]
lgtm.

**Pull request merged**